types/RegistrationType.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "types/RegistrationType",
+    "type": "object",
+    "allOf": [
+        {"$ref": "../types/IdentifierType.json"}
```

Comment:
I assume because this just inherits from `IdentifierType` and there's no extra fields that we're doing this for naming reasons?

Reply:
Yes, naming reasons (documentation) only. Have put explanation in RegistrationType file now.